source/support.tex

```diff
@@ -209,30 +209,30 @@
 
   // \ref{alg.c.library}, C standard library algorithms
   void* bsearch(const void* key, const void* base, size_t nmemb, size_t size,
-                @\placeholder{c-compare-pred}@*@\itcorr[-1]@ compar);
+                @\placeholder{c-compare-pred}@*@\itcorr[-1]@ compar);                                // freestanding
   void* bsearch(const void* key, const void* base, size_t nmemb, size_t size,
-                @\placeholder{compare-pred}@*@\itcorr[-1]@ compar);
-  void qsort(void* base, size_t nmemb, size_t size, @\placeholder{c-compare-pred}@*@\itcorr[-1]@ compar);
-  void qsort(void* base, size_t nmemb, size_t size, @\placeholder{compare-pred}@*@\itcorr[-1]@ compar);
+                @\placeholder{compare-pred}@*@\itcorr[-1]@ compar);                                  // freestanding
```

Comment:
I think it's preferable to have these comments
be on the same line as the declaration's identifier they apply to.

Reply:
I think I've fixed this.